Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -785,4 +794,24 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         alert.addAction(.init(title: L10n.Alert.Actions.ok, style: .default, handler: { _ in }))
         present(alert, animated: true)
     }
+    
+    // MARK: - InputTextViewImagePasteDelegate
+    
+    open func inputTextView(_ inputTextView: InputTextView, didPasteImage image: UIImage) {
+        do {
+            let imageUrl = try image.temporaryLocalFileUrl()
```

Comment:
The function is throwing and it still returns an optional URL?

Reply:
Yes. It throws in case of a write failure but returns optional in case we could not create a jpeg representation of the image.